+sw_tests/+unit_tests/unittest_spinw_spec2MDHisto.m

```diff
@@ -0,0 +1,39 @@
+classdef unittest_spinw_spec2MDHisto < sw_tests.unit_tests.unittest_super
+    properties
+    end
+    methods (Test)
+        function test_1_0_0(testCase)
+            q0 = [0 0 0];
+            qdir = [1 0 0];
+            spec = sw_egrid(spinwave(sw_model('triAF', 1), {q0 q0+qdir nsteps}));
```

Comment:
The spinw model appears to be the same used for all tests. You can add this to a `TestClassSetup` or `TestMethodSetup` like so
https://github.com/SpinW/spinw/blob/49880bc38bc853424a8cee431ce36fb5cdde4b55/%2Bsw_tests/%2Bunit_tests/unittest_spinw_optmagk.m#L16-L29
In this case `TestClassSetup` is probably best as it isn't changed in any of the tests.